ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/views/taskButton/TaskButtonCell.css

```diff
@@ -5,6 +5,26 @@
  *
  * http://www.sencha.com/products/gxt/license/
  */
+@font-face {
+  font-family: TextaRegular;
+  src: url(../../Texta_Font/Texta-Regular.otf);
+}
+
+@font-face {
+  font-family: TextaBlack;
+  src: url(../../Texta_Font/Texta-Black.otf);
+}
+
+@font-face {
+  font-family: TextaBold;
+  src: url(../../Texta_Font/Texta-Bold.otf);
+}
+
+@font-face {
+  font-family: TextaItalic;
+  src: url(../../Texta_Font/Texta-BlackItalic.otf);
+}
```

Comment:
This is a really general thing to have in what seems to be a really specific CSS file. Especially since this file itself only uses TextaRegular; and, I've seen other places which reference TextaBold without having it in their own file. I don't know enough about how GWT does things to really know how this CSS is delivered, but ideally it should either just be only as much as needed per file (if they're delivered independently, on the fly) or only once overall (if they're all put together).